src/App/Kernel.php

```diff
@@ -372,7 +390,18 @@ public function routes(Application $app)
 
     public function handleException($e): Response
     {
-        return $response;
+        /** @var LoggerInterface $logger */
+        $logger = $this->get('logger');
+        $logger->critical('An unhandled exception was thrown', [
+            'exception' => $e,
```

Comment:
nice to have `exception` in the context, albeit `critical` is a bit too much, `error` is the level we have standardized on for single HTTP requests failing